requirements-dev.txt

```
@@ -1,9 +1,14 @@
+black==22.10.0
 build==0.9.0
 codecov==2.1.12
+flake8==3.9.2
 invoke==1.7.3
+isort==5.10.1
 jsondiff==2.0.0
+mypy==0.981
```

A newer version of mypy is out.
```suggestion
mypy==0.991
```